### DISCUSSION TITLE
Trouble installing libzmq3-dev on GitHub Actions workflows that use clustermq

### DISCUSSION THREAD

**Discussion by wlandau** [mschubert/clustermq#262] [Q&A] [answered]
This is not explicitly related to the `clustermq` code base, but it is affecting `targets` because of the dependency on ZeroMQ through `clustermq`. I am having trouble installing libzmq3-dev on Ubuntu 18.04 on GitHub Actions. An example of the full error is at https://github.com/ropensci/targets/runs/2545335843#step:8:124, and I created a more minimal reproducible example at https://github.com/wlandau/libzmq-reprex. Switching to Ubuntu 20.04 does not work because of different errors: https://github.com/ropensci/targets/runs/2544918909?check_suite_focus=true#step:8:102.

**Comment by mschubert** [comment, marked as answer]
Maybe this helps? https://github.com/ropensci/rzmq/blob/master/.github/workflows/R-CMD-check.yaml

**Reply by wlandau** [reply]
Thanks @mschubert, I switched to Ubuntu 20.04 and it seems to work now.
